tests/unit/tf/examples/test_01_getting_started.py

```diff
@@ -42,12 +42,15 @@ def test_example_01_getting_started(tb):
     metrics = tb.ref("metrics")
     assert set(metrics.keys()) == set(
         [
-            "auc",
-            "binary_accuracy",
             "loss",
-            "loss_batch",
+            "rating/regression_output_loss",
```

Comment:
why is there a regression output here? getting-started is for binary classification task. is there a reason we have regression in the output metrics?